conductr_cli/host.py

```diff
@@ -43,4 +44,55 @@ def with_docker_machine():
 
 
 def loopback_device_name():
-    return 'lo' if platform.system().lower() == 'linux' else 'lo0'
+    return 'lo' if is_linux() else 'lo0'
+
+
+def is_linux():
+    return platform.system().lower() == 'linux'
+
+
+def is_osx():
+    return platform.system().lower() == 'darwin'
+
+
+def can_bind(ip_addr, port):
+    socket_af = socket.AF_INET if ip_addr.version == 4 else socket.AF_INET6
+    s = socket.socket(socket_af, socket.SOCK_STREAM)
+
+    result = False
+    try:
+        s.bind((ip_addr.exploded, port))
+        result = True
+    except OSError:
+        pass
+    finally:
+        s.close()
+
+    return result
+
+
+def addr_alias_setup_instructions(addrs, subnet_mask):
```

Comment:
In the output, we should substitute the network aliases that already have been created.

**Current output**

```
sudo ifconfig lo0 alias 192.168.1.1 255.255.255.0
sandbox run 2.0.0 -n 3
Error: Unable to allocate bind address
Error: Run the following commands to create address alias on your machine:

sudo ifconfig lo0 alias 192.168.1.2 255.255.255.0
sudo ifconfig lo0 alias 192.168.1.3 255.255.255.0
sudo ifconfig lo0 alias 192.168.1.4 255.255.255.0
```

**Expected output**

```
sudo ifconfig lo0 alias 192.168.1.1 255.255.255.0
sandbox run 2.0.0 -n 3
Error: Unable to allocate bind address
Error: Run the following commands to create address alias on your machine:

sudo ifconfig lo0 alias 192.168.1.2 255.255.255.0
sudo ifconfig lo0 alias 192.168.1.3 255.255.255.0
```